tests/bash_tap_fd.sh

```diff
@@ -10,6 +10,10 @@ function get_script {
     echo "/usr/bin/env python ${bashtap_org_pwd}/../firefox_decrypt.py"
 }
 
+function get_script_python3 {
```

Comment:
This isn't necessary, use `get_script_python` instead.
We run tests on different machines. One has python 2 as default others have python 3 so `python` can either be 2 or 3 depending on where tests run.